okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -93,6 +98,21 @@
   private static final Logger log = Logger.getLogger(OkHttpClientTransport.class.getName());
   private static final OkHttpClientStream[] EMPTY_STREAM_ARRAY = new OkHttpClientStream[0];
 
+  private static final Resource<ExecutorService> SHARED_EXECUTOR =
+      new Resource<ExecutorService>() {
+        @Override
+        public ExecutorService create() {
+          return Executors.newCachedThreadPool(new ThreadFactoryBuilder()
```

Comment:
Question:  Cached thread pools can easily create a lot of threads.  Since this will be used in multiple contexts (i.e. ones outside of this class), would it be a good idea to limit them?